app/_hub/kong-inc/statsd/_index.md

```diff
@@ -87,49 +81,119 @@ params:
       required: true
       default: All metrics are logged
       datatype: Array of record elements
-      description: 'List of Metrics to be logged. Available values are described under [Metrics](#metrics).'
+      description: |
+        List of metrics to be logged. Available values are described under [Metrics](#metrics).
     - name: prefix
       required: true
       default: '`kong`'
       datatype: string
       description: String to prefix to each metric's name.
+    - name: hostname_in_prefix
+      required: true
+      default: '`false`'
+      datatype: boolean
+      description: Include the `hostname` in the `prefix` for each metric name.
+    - name: udp_packet_size
+      required: true
+      default: '`0` (not combined)'
+      datatype: number
+      description: |
+        Combine UDP packet up to the size configured. If zero (0), don't combine the
+        UDP packet. Must be a number between 0 and 65507 (inclusive).
+    - name: use_tcp
+      required: true
+      default: '`false`'
+      datatype: boolean
+      description: Use TCP instead of UDP.
+    - name: allow_status_codes
+      required: true
+      default: All responses are passed to log metrics
+      value_in_examples:
+        - 200-205
+        - 400-499
+      datatype: array of string elements
+      description: List of status code ranges that are allowed to be logged in metrics.
+    - name: consumer_identifier_default
+      required: true
+      default: 'custom_id'
+      datatype: string
+      description: The default consumer identifier of metrics. This takes effect when a metric's consumer identifier is omitted. Allowed values are `custom_id`, `consumer_id`, `username`.
+    - name: service_identifier_default
+      required: true
+      default: 'service_name_or_host' 
+      datatype: string
+      description: The default service identifier of metrics. This takes effect when a metric's service identifier is omitted. Allowed values are `service_name_or_host`, `service_id`, `service_name`, `service_host`.
+    - name: workspace_identifier_default
+      required: true
+      default: 'workspace_id' 
+      datatype: string
+      description: The default workspace identifier of metrics. This will take effect when a metric's workspace identifier is omitted. Allowed values are `workspace_id`, `workspace_name`.   
+    - name: service_prefix_in_metric
+      required: false
+      default: false 
+      datatype: boolean
+      description: | 
+      Indicate whether to add `service.` prefix before `<service_identifier>` in metric name. 
+      
+      For example, if set to `true`, then `kong.<service_identifier>.request.count` will be `kong.service.<service_identifier>.request.count`.
```

Comment:
```suggestion
          Indicate whether to add `service.` prefix before `<service_identifier>` in metric name. 
          
          For example, if set to `true`, then `kong.<service_identifier>.request.count` will be `kong.service.<service_identifier>.request.count`.
```
Missed an indent, this will fix the failing build.